drivers/clock_control/clock_control_mcux_ccm.c

```diff
@@ -377,6 +377,15 @@ static int mcux_ccm_get_subsys_rate(const struct device *dev,
 					/ (CLOCK_GetDiv(kCLOCK_Flexio2Div) + 1);
 	} break;
 #endif
+
+#ifdef CONFIG_ADC_MCUX_LPADC
+	case IMX_CCM_LPADC1_CLK:
+		*rate = CLOCK_GetRootClockFreq(kCLOCK_Root_Adc1);
+		break;
+	case IMX_CCM_LPADC2_CLK:
+		*rate = CLOCK_GetRootClockFreq(kCLOCK_Root_Adc2);
```

Comment:
do all platforms have these two enums defined ? ie what if a platform has only 1 adc, will it cause a build failure, or do we have no cases like that yet (if not, then it's fine for now)

Reply:
Currently, for all platforms equipped with LPADC and CCM controllers(RT116x, RT117x), the number of ADC instances is 2.